catalog/dags/common/loader/reporting.py

```diff
@@ -166,7 +167,7 @@ def report_completion(
         media_type_reports += "\n"
 
     date_range = "_all_"
-    if dated:
+    if dated and not is_reingestion_workflow:
         date_range = f"{date_range_start} -> {date_range_end}"
```

Comment:
I might recommend actually making this a slightly different message rather than leaving it as `_all_`:
```suggestion
    if dated:
        date_range = f"{date_range_start} -> {date_range_end}"
    if is_reingestion_workflow:
        date_range = "_multi-time period spread_"
```

Reply:
Heads up, looks like this still needs to be changed!